package.json

```diff
@@ -21,11 +21,11 @@
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
-    "@theforeman/builder": "^8.16.0",
-    "@theforeman/eslint-plugin-foreman": "^8.16.0",
-    "@theforeman/stories": "^8.16.0",
-    "@theforeman/test": "^8.16.0",
-    "@theforeman/vendor-dev": "^8.16.0",
+    "@theforeman/builder": "^10.1.0",
```

Comment:
The core pr will be probably merged before this one https://github.com/theforeman/foreman/pull/9110

Reply:
The core PR was indeed merged before this one, resolving.